### ISSUE TITLE
Disable Animation

### ISSUE THREAD

**Issue by jaeaton** [beto-rodriguez/LiveCharts2#45]
When plotting real time data fast the animation makes the plot hard to follow. IE if you change the wait duration of the autoupdate example to 100 ms the left hand side of the chart goes squirrely and it is difficult to get a good reading on the values being plotted.

Ideally there would be a feature to disable animation or set the animation time to 0.

**Comment by vadimffe** [comment]
I agree! I have not tested version 2, but in first version it does not look nice when you have refresh rate of 10 seconds and charts are flying towards user all the time. I think animation should appear once you open an application for the first time, but not while passing data to chart. For example I would prefer Pie chart to just smoothly resize sections, instead of blinking (read it as animation) constantly. Good example is how Excel charts are working, smooth.
In version 1 if you set animation to disabled charts are "blinking" anyway. 
At least this should be configurable.

![1735D63B-5308-434E-8AF0-2973B68B5547](https://user-images.githubusercontent.com/72302395/119847774-f0ca3680-bf13-11eb-883f-09cad43b79d1.GIF)

**Comment by vadimffe** [comment]
I have not tested version 2, but in first version it does not look nice when you have refresh rate of 2 seconds and charts are "flying towards user" all the time. I think animation should appear once you open an application for the first time, but not while passing data to chart. For example I would prefer Pie chart to just smoothly resize sections, instead of blinking (read it as animation) constantly. Good example is how Excel charts are working, smoothly resizing sections on values change.
In version 1 if you set animation to disabled charts are "blinking" anyway with `DisableAnimations="True"`.
At least this should be configurable.

![1735D63B-5308-434E-8AF0-2973B68B5547](https://user-images.githubusercontent.com/72302395/119849589-6d114980-bf15-11eb-8f66-a0133a829194.GIF)

**Comment by beto-rodriguez** [comment]
OK, I am building docs right know and will make an article to animations, you are getting it wrong @vadimffe, v2 has smoother animations, and it has a much better performance.

I will build an article about animations, and will keep this issue open to let you know when it is ready, but in general here are some points you are getting wrong:

You can disable animations in v2, all the charts have the `AnimationsSpeed` and `EasingFunction` properties, play with them and you will achieve your goal, for example I can disable anumations settings the `EasingFunction` property to null.

``` c#
<lvc:CartesianChart Series="{Binding Series}"
                                AnimationsSpeed="0:0:5"
                                EasingFunction="{x:Null}">        
</lvc:CartesianChart>
```

V2 now introduces many new animations, and you can create your own, but you can make the chart bounce or follow any easing curve:

![image](https://user-images.githubusercontent.com/10853349/119852559-0409e180-bed5-11eb-801b-a321d00ad3ec.png)

@jaeaton changing the animations speed could work in your case, but I think that the library should expose a property called maybe "UpdaterThrottler" where we can let the chart know a `TimeSpan` at what rate it should update the UI, this property exists and it is 10ms now, we should just expose it so everyone can avoid this "sync" errors.

**Comment by beto-rodriguez** [comment]
I am closing this for now, you can now have more information at:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/docs/1.overview/1.9.animations.md

The article will be easier to read when the new site is published
